### DISCUSSION TITLE
How to increase Code block font?!

### DISCUSSION THREAD

**Discussion by zambala** [Akifyss/obsidian-border#112] [Q&A] [answered]
Your theme is great!

But how to increase the font size for Code block?!

The font-size for Code block is critically too small, especially on a dark background....

I have found this - but it works only in Reading View....

```
.markdown-preview-view code {
  font-family: Courier New !important;
  font-size: 18px !important;
}
```

How to do the same for Preview / Source Views ?!

**Comment by Akifyss** [comment, marked as answer]
try this!
```css
body{
--code-size:18px;
}
```

**Reply by zambala** [reply]
Thanks! This is much better now!

**Comment by gpfriend** [comment]
Newbie here: Where do I put this code? Thanks!
